docs/README_NPM.md

```
@@ -0,0 +1,241 @@
+### 🆕 CLI Available
+
+> - **Run via npx (no installation required)**
+>   - Requirements: NodeJS LTS v24 or later
+>   - Run `npx hili-lipsum --help`<br><br>
+> - **Docker image**
+>   - A Docker image is available at https://hub.docker.com/r/weaponsforge/hili-lipsum
+
+### Content
+
+- [Content](#content)
+- [Quick Start](#-quick-start)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)